app/controllers/api/v2/provisioning_templates_controller.rb

```diff
@@ -0,0 +1,133 @@
+module Api
+  module V2
```

Comment:
should we really create a new controller rather then introduce a change in routes?